components/crud-web-apps/volumes/Dockerfile

```diff
@@ -21,11 +22,18 @@ RUN npm run build
 FROM node:12-buster-slim as frontend
 
 WORKDIR /src
-COPY ./volumes/frontend/package*.json ./
-RUN npm install
-COPY --from=frontend-kubeflow-lib /src/dist/kubeflow/ ./node_modules/kubeflow/
 
-COPY ./volumes/frontend/ .
+COPY ./volumes/frontend/package.json ./
+COPY ./volumes/frontend/package-lock.json ./
+COPY ./volumes/frontend/tsconfig.json ./
+COPY ./volumes/frontend/tsconfig.app.json ./
+COPY ./volumes/frontend/tsconfig.spec.json ./
+COPY ./volumes/frontend/angular.json ./
+COPY ./volumes/frontend/src ./src
+
+ENV NG_CLI_ANALYTICS "ci"
```

Comment:
Seems like Line14 and Line34 are the same.

Reply:
Since this is a multi-stage docker build will the ENV var persist between stages?

Reply:
Oh that's multi-stage, I didn't notice at the beginning.